index.html

```diff
@@ -41,8 +41,171 @@ <h3 class="connect_me">
+                    <li class="tag-item">html</li>
+                    <li class="tag-item">css</li>
+                    <li class="tag-item">javaScript</li>
+                </ul>
+                <button class="action-button">
+                    <h4 class="button-text">See Project</>
+                </button>
+            </div>
 
+            <div class="card">
+                <img class ="card-img" src="assits/Snapshoot-Portfolio4.png" alt="The Snapshoot-Portfolio image">
+                <div class="main-card-text">
+                    <h1 class="card-title">Multi-Post</br>
+                        Stories</h1>
+                    <div class="card-description">
+                        <ul class="card-desc">
+                            <li class="client-text">CANOPY</li>
+                            <li class="role-text">&#9679;  BACK END DEV</li>
+                            <li class="year-text">&#9679;  2015</li>
+                        </ul>
+                    </div>
+                </div>
+                <P class="card-info">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </P>
+                <ul class="tags">
+                    <li class="tag-item">html</li>
+                    <li class="tag-item">css</li>
+                    <li class="tag-item">javaScript</li>
+                </ul>
+                <button class="action-button">
+                    <h4 class="button-text">See Project</>
+                </button>
+            </div>
+
+        </section>
+
+
+        <section class="about-me">
+            <div class="about-me-intro">
+                <div class="container">
+                    <h1 class="welcome">
+                        About <br/> Myself
+                    </h1>
+                    <p class="about_me">
+                        Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.                    </p>
+                    <h3 class="connect_me">
+                        <a href="#">LET'S CONNECT</a>
+                    </h3>
+                    <ul class="icon-group">
+                        <li class="icon"> <a href="https://github.com/qusai122"><i class="fa-brands fa-github"></i></a></li>
+                        <li class="icon"> <a href="https://www.linkedin.com/in/qusai-issa-ab5a59156/"><i class="fa-brands fa-linkedin"></i></a></li>
+                        <li class="icon"> <a href=""><i class="fa-brands fa-angellist"></i></a></li>
+                        <li class="icon"> <a href=""><i class="fa-brands fa-twitter"></i></a></li>
+                        <li class="icon"> <a href="mailto:qusai.b.issa@gmail.com"><i class="fa-brands fa-m"></i></a></li>
+                    </ul>
+                    <button class="action-button">
```

Comment:
- [ ] Please follow the Figma design you are not following the Figma design. Kindly fix your about section also check below :point_down: 

You design:
![Screenshot from 2023-02-19 00-43-13](https://user-images.githubusercontent.com/45226848/219889897-bbcc741a-fdbd-4684-8db8-99511b5db456.png)

Figma Design:
![Screenshot from 2023-02-19 00-42-49](https://user-images.githubusercontent.com/45226848/219890214-7bde4991-7efb-4210-b925-32dfe19bf352.png)